CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -370,7 +370,7 @@ function Panel:EvaluateGaze(cameraCF, cameraRenderCF, userHeadCF, lookRay)
 		--transform worldIntersectPoint to gui space
 		local gui = self:GetGUI()
 		local guiWidth, guiHeight = gui.AbsoluteSize.X, gui.AbsoluteSize.Y
-		local localIntersectPoint = planeCF:pointToObjectSpace(worldIntersectPoint) * Vector3.new(-1, 1, 1) + Vector3.new(self.width / 2, -self.height / 2, 0)
+		local localIntersectPoint = (planeCF:pointToObjectSpace(worldIntersectPoint) / currentHeadScale) * Vector3.new(-1, 1, 1) + Vector3.new(self.width / 2, -self.height / 2, 0)
```

Comment:
I think you'll have worse things happen to you if it can. But I don't think so, no.